robottelo/hosts.py

```diff
@@ -716,7 +716,10 @@ def configure_rhai_client(
         if rhel_repo is None:
             raise ContentHostError(f'Missing RHEL repository configuration for {rhel_distro}.')
 
-        self.create_custom_repos(**{rhel_distro: rhel_repo})
+        if rhel_distro == constants.DISTRO_RHEL8:
```

Comment:
This is going to be the same for RHEL9 and so forth. We should reverse the logic and make sure the rhel_distro is not `constants.DISTRO_RHEL6` nor `constants.DISTRO_RHEL7`

Reply:
@rplevka updated the pr.

Reply:
thanks